src/juicy-popover.html

```diff
@@ -22,7 +22,8 @@
             position: absolute;
             top: 0;
             left: 0;
-            z-index: 999;
+            z-index: 999; /*For IE11 support*/
```

Comment:
Please remove this because we use ES6 all over the element. It doesn't support IE in the first place.